GoogleDataTransport/GDTCORLibrary/Public/GDTCOREvent.h

```diff
@@ -63,6 +62,10 @@ typedef NS_ENUM(NSInteger, GDTCOREventQoS) {
 /** The clock snapshot at the time of the event. */
 @property(nonatomic) GDTCORClock *clockSnapshot;
 
+/** The fileURL of clearDataObjectTransportBytes in disk. set when store the transport bytes to
+ * disk,*/
+@property(nonatomic) NSURL *fileURL;
```

Comment:
I'd suggest making this readonly so it can only be set by calling `-writeToURL:` (a method you should write, see comment above)